Screen Reader and Keyboard Accesibility

msrafi opened the pull request:
Added Following Functionality

1. Screen Reader Attributes
2. Keyboard Accessibility
3. Focus Handling in Auto play mode

gustavpursche commented:
@kenwheeler Are you somehow willing to merge this in? Is there something needed?

kenwheeler commented:
I am I just need to review it

kenwheeler commented:
@msrafi can you rebase?

msrafi commented:
@kenwheeler <https://github.com/kenwheeler> Ohhh, created a new pull
request.

On Tue, Mar 10, 2015 at 10:23 AM, Ken Wheeler <notifications@github.com>
wrote:

> @msrafi <https://github.com/msrafi> can you rebase?
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kenwheeler/slick/pull/838#issuecomment-78063037>.
>

msrafi commented:
@kenwheeler http://jsfiddle.net/1x8w9c3n/31/

umpirsky commented on slick/slick.js:
```diff
@@ -481,9 +481,9 @@
-        if (_.options.accessibility === true) {
-            _.$list.prop('tabIndex', 0);
-        }
+        // if (_.options.accessibility === true) {
```
Should be removed?

msrafi replied:
Yes. Let me create another fiddle with the latest code.

msrafi commented:
Updated with later version: http://jsfiddle.net/equinoxs/1x8w9c3n/39/ 
About this one:
```javascript
     if (_.options.accessibility === true) {
              _.$list.prop('tabIndex', 0);
       }
```

This actually removed because, the tabIndex is handled inside _.activateADA method to give tabIndex to the active slide instead of all slides.

Fixed one more bug:
//Dont slide if the cursor is inside the form fields and arrow keys are pressed

simeydotme commented:
looks like this was fixed by #1467 ?
